src/Microsoft.Azure.EventHubs/Primitives/ClientInfo.cs

```diff
@@ -23,7 +23,7 @@ static ClientInfo()
                 product = GetAssemblyAttributeValue<AssemblyProductAttribute>(assembly, p => p.Product);
                 version = GetAssemblyAttributeValue<AssemblyVersionAttribute>(assembly, v => v.Version);
                 framework = GetAssemblyAttributeValue<TargetFrameworkAttribute>(assembly, f => f.FrameworkName);
-#if NETSTANDARD
+#if NETSTANDARD1_3
```

Comment:
Sean, I see what you mean know after a second thought. I will sign off the change. Thx for pointing this.